ci/deps/actions-310-numpydev.yaml

```diff
@@ -21,6 +26,7 @@ dependencies:
 
   - pip:
     - "cython"
+    - "git+https://github.com/mesonbuild/meson-python.git@e74218846ab386faa761558987064508cbf6a28e"
```

Comment:
Is this supposed to be both in the pip section and in the standard dependencies?

Reply:
related comment: no need to install from GitHub anymore; `meson-python` 0.13.1 should resolve all outstanding issues and was released last week.

Reply:
So sorry for the noise. I was changing these files by hand, and forgot to delete the old pinned meson-python. I'll update ASAP.